Documentation/Attacks.md

```diff
@@ -0,0 +1,147 @@
+# Attacks
+
+## Types ##
+
+### Direction ###
+An integer from 0 to 3. 0 is East, each increment is 90 degrees clockwise.
+
+### Speed ###
+Pixels per second. Multiply an original Undertale value with 30 (FPS) to get its equivalent.
+
+### Color ###
+0 for white, 1 for blue. Other colors (such as orange) may or may not be added in the future
+
+### Time ###
+Seconds
+
+### Angle ###
+Degrees
+
+### Boolean ###
+Just an integer 0 or 1
+
+### Count ###
+How many
+
+### Spacing ###
+Pixels
+
+<hr />
+
+### BoneH ###
+Creates a horizontal bone
+
+* XPosition
+* YPosition
+* Width
+* Direction
+* Speed
+* Color
+
+### BoneV ###
+Creates a vertical bone
+
+* XPosition
+* YPosition
+* Height
+* Direction
+* Speed 
+* Color
+
+### BoneHRepeat ###
+Creates many horizontal bones
+
+* StartX
+* StartY
+* Width
+* Direction
+* Speed
+* Count
+* Spacing
+
+### BoneVRepeat ###
+Creates many vertical bones
+
+* StartX
+* StartY
+* Height
+* Direction
+* Speed
+* Count
+* Spacing
+
+### SineBones ###
+> Created out of laziness, probably shouldn't use.
+
+<!-- TODO: What it does? -->
```

Comment:
It creates a series of bones that leaves a sine-wave gap in between them.

Reply:
Sorry, I think I forgot the TODO